### PR TITLE
Enable Travis-CI

### PR DESCRIPTION
**Pull request by kevmoo**
Fixes https://github.com/dart-lang/string_scanner/issues/6

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+dart:
+  - dev
+  - stable
+
+dart_task:
+  - test: --platform vm
+  # No parallelism on Firefox (-j 1)
+  # Causes flakiness – need to investigate
+  - test: --platform firefox -j 1
```
Comment (nex3) [inline .travis.yml]: Why Firefox? I thought most of our other packages used Chrome.

The tests on this package are fast enough that it's probably more efficient to run browser and VM tests in the same job.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+language: dart
+
+dart:
+  - dev
+  - stable
+
+dart_task:
+  - test: --platform vm
+  # No parallelism on Firefox (-j 1)
+  # Causes flakiness – need to investigate
```
Comment (nex3) [inline .travis.yml]: This should have an assigned TODO.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  - test: --platform firefox -j 1
+  - dartanalyzer
```
Comment (nex3) [inline .travis.yml]: I'd put this under `matrix.include` as well. I don't think there's likely to be a lot of value in double-testing analysis.

Comment (kevmoo) [reply .travis.yml]: As long as stable is supported, I think it's good to track – help find potential API changes that are not backwards compatible.

Comment (nex3) [reply .travis.yml]: We already get that from testing on stable. This package isn't doing anything statically complex enough to be worth the extra job time from this task.

Comment (kevmoo) [reply .travis.yml]: ack

**Pull request merged**